build.fsx

```diff
@@ -0,0 +1,302 @@
+
+    let runWithDefaultOptions framework projFile args =
+        run id framework projFile args
+
+let pathToSpec =
+    "src/Generators/Generator.Bind/Specifications"
+
+let specSource =
+    "https://raw.githubusercontent.com/frederikja163/OpenGL-Registry/master/xml/gl.xml"
+
+//let bindingsOutputPath =
+//    ""
+
+let asArgs args =
+    args |> String.concat " "
+
+Target.create "UpdateBindings" (fun _ ->
+    Trace.log " --- Updating bindings --- "
+    let framework = "netcoreapp20"
+    let projFile = "src/Generators/Generator.Bind/Generator.Bind.csproj"
+    let args =
+        [ sprintf "-i %s" (System.IO.Path.GetFullPath pathToSpec)
+          "-a Desktop"
+        ] |> asArgs
+    DotNet.runWithDefaultOptions framework projFile args
+    |> ignore
+)
+
+
+Target.create "UpdateSpec" (fun _ ->
+    Trace.log " --- Updating spec --- "
+    let framework = "netcoreapp20"
+    let projFile = "src/Generators/Generator.Converter/Generator.Convert.csproj"
+    let relativePathToSpecification = "OpenGL/GL/4.6"
+    let fetchSpec relativePathToSpecification =
+        let relativePathToSpecificationFile = relativePathToSpecification </> "specification.xml"
+        let fullPathToSpecificationFile = System.IO.Path.GetFullPath(pathToSpec </> relativePathToSpecificationFile)
+        Directory.create (pathToSpec </> relativePathToSpecification)
+        let args =
+            [   sprintf "-i %s" specSource
+                sprintf "-o %s" fullPathToSpecificationFile
+                "--prefix gl"
+            ] |> asArgs
+        DotNet.runWithDefaultOptions framework projFile args
+        |> ignore
+    [ "OpenGL/GL/4.6"
+      "OpenGL/ES/3.2" ]
+    |> List.iter fetchSpec
+)
+
+// ---------
+// Build Targets
+// ---------
+
+Target.create "Clean" (fun _ ->
+    Shell.cleanDir binDir
+)
+
+Target.create "Restore" (fun _ ->
+     Shell.Exec("dotnet", "restore OpenTK.sln") |> ignore
+ )
+
+// Generate assembly info files with the right version & up-to-date information
+Target.create "AssemblyInfo" (fun _ ->
+    //TODO: Create and update the Directory.Build.props file with the version information taken from the releaseNotes value.
+    // see https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#directorybuildprops-and-directorybuildtargets
+    Trace.traceError "Unimplemented."
+)
+
+
+
+Target.create "Build" (fun _ ->
+    releaseProjects
+    |> MSBuild.runRelease id "" "Build"
+    |> Trace.logItems "Build-Output: "
+)
+
+Target.create "BuildTest" (fun _ ->
+    !! "tests/**/*.??proj"
+    |> MSBuild.runDebug id testDir "Build"
+    |> Trace.logItems "TestBuild-Output: "
+)
+
+
+// Copies binaries from default VS location to expected bin folder
+// This preserves subdirectories.
+Target.create "CopyBinaries" (fun _ ->
+    releaseProjects
+    |>  Seq.map (fun f -> ((System.IO.Path.GetDirectoryName f) @@ "bin/Release/netstandard2.0", "bin" @@ (System.IO.Path.GetFileNameWithoutExtension f)))
```

Comment:
Would it make sense, to put that in a variable, for easier maintenance..... `/netstandard2.0` i mean (Im not sure if the review thing here, keeps the thing i marked)